aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1485,10 +1485,13 @@ void scaled_gemm(
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_A_SCALE_POINTER, mat1_scale_ptr);
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_B_SCALE_POINTER, mat2_scale_ptr);
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_D_SCALE_POINTER, result_scale_ptr);
+#if defined(USE_ROCM) && ROCM_VERSION >= 60200
```

Comment:
Does this preserve the semantics of the non-ROCM use-case?

Reply:
Nope! My mistake, leftover from debug. I will fix